doc/tutorials/acrn-sanity-test.rst

```diff
@@ -0,0 +1,227 @@
+ACRN hypervisor’s availability. User could manually run these after setup ACRN. 
+These cases can be implemented on ``kbl-nuc-i7``.
+
+ACRN Sanity Test Cases
+######################
+
+Hypervisor
+##########
+
+Introduction
+************
+
+ST_HV_Bootloader_LaaG_VUEFI_Default
+***********************************
+
+* Description
+
+The test case focuses on testing boot up User VM using a virtual UEFI (OVMF) firmware and the default ``launch_uos.sh``.
+
+* Prerequisites
+
+Service VM boots up normally
+
+* Test Steps
+
+1. Launch Service VM
+#. Use ``OVMF.fd`` to launch clearlinux using the default ``launch_uos.sh`` script
+
+  .. code-block:: none
+
+     $ mkdir -p /home/<your user name>/uos/
+
+     $ cd /home/<your user name>/uos/
+
+     $ curl https://download.clearlinux.org/releases/xxxxx/clear/clear-xxxxx-kvm.img.xz -o uos.img.xz
+
+     $ unxz uos.img.xz
+
+     $ sudo sed -i "s|/home/clear|$HOME|g" /usr/share/acrn/samples/nuc/launch_uos.sh
+
+     $ sudo /usr/share/acrn/samples/nuc/launch_uos.sh
+
+* Expected Result
+
+User VM boot up normally
+
+ST_HV_Memory_LaaG_Default_Partition_Check
+*****************************************
+
+* Description
+
+This test case focus on checking if the pass through memory to User VM and
+check if the memory size is in a reasonable scope as ``launch_uos.sh`` defined.
+Since the User VM will take a part of memory, the available memory size will
+not be exact as what pass-through.
+
+* Prerequisites
+
+Service VM and User VM boot up normally
+
+* Test Steps
+
+1. Boot to Service VM
+#. Boot User VM with default memory setting in ``launch_uos.sh``
+#. Check ``cat /proc/meminfo`` and see if the available memory is close the size passed through to User VM
+
+Expected Result
+
+1. Boot User VM successfully
+#. memory is close the size passed through to User VM
+
+IO
+**
+
+ST_ACRNIO_LaaG_Virtio_net
+*************************
+
+* Description
+
+This test case focus on checking if Service VM and User VM can be allocated different IP,
+and ping is ok by sharing same physical network card.
+
+* Prerequisites
+
+VirtIO and bridge/tap is created normally in Service VM
+
+* Test Steps
+
+1. Boot the Device Under Test (DUT), and enter Service VM from the ACRN hypervisor console (serial port).
+
+   .. code-block:: none
+
+      ACRN:\>vm_console
+
+#. Check IP and bridge/tap after log in Service VM.
+
+   .. code-block:: none
+     
+      $ ip a
+      $ brctl show
+      bridge name     bridge id               STP enabled     interfaces
+      acrn-br0         8000.1ad4ac453b20       no              acrn_tap0
+                                                               ens4
+
+#. Boot User VM using the ``launch_uos.sh`` and check IP address.
+
+   .. code-block:: none
+   
+      $ sudo /usr/share/acrn/samples/nuc/launch_uos.sh
+
+      $ ip a
+
+* Expected Result
+
+IP address should be allocated for both Service VM  and User VM, and ping is ok
+
+
+ST_ACRNIO_SOS_Storage
```

Comment:
I still do not understand this test at all. You want to test the storage device you've just booted from by mounting it read/write for a second time (since it's already mounted under `/`)... I can't make any sense of this.

Reply:
We want to test it can check the storage when plug new one

Reply:
OK, if it's about testing access to a storage device other than the one hosting the rootfs, then please make this specific.